src/sentry/static/sentry/app/components/similarSpectrum.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
I think we need to start introducing some subfolders inside `/components` to help organize these things (I feel like there's 100+ files in `/components/*`).

Reply:
Yeah, alternatively we may want to think structure on a broader level (i.e. folders per feature). Burying components within subfolders doesn't necessarily help. I'm guessing many of the components inside of `/components/` are only used once.